### DISCUSSION TITLE
Per-process Statistics min/max/difference

### DISCUSSION THREAD

**Discussion by po0p** [winsiderss/systeminformer#1762] [Questions / Answers]
Hey, how do i get those to show? 
![image](https://github.com/winsiderss/systeminformer/assets/68482185/07a14274-c6a4-4551-92fa-3b186c75130f)
Are they just collected at all? I have been looking at that window for solid 5 minutes and only have real-time data. Is there some setting I am missing?

**Comment by dmex** [comment]
> Are they just collected at all?

No, this feature hasn't been enabled/functional yet.
